src/main/java/de/dennisguse/opentracks/viewmodels/ClockViewHolder.java

```diff
@@ -22,6 +22,7 @@ public void configureUI(DataField dataField) {
     }
 
     @Override
-    public void onChanged(UnitSystem unitSystem, RecordingData data) {
+    public void onChanged(UnitSystem unitSystem, RecordingData data) throws UnsupportedOperationException {
+        // We don't have enough information or context to provide a complete implementation in the base class or interface for this methos. Therefore, this is empty.
```

Comment:
You need to throw exception inside this method

Reply:
Okay.